.env.example

```diff
@@ -2,6 +2,10 @@ APP_ENV=local
 APP_DEBUG=true
 APP_KEY=SomeRandomString
 
+NORTHSTAR_URL=http://northstar.app:8000
+NORTHSTAR_API_KEY=abc4324
+NORTHSTAR_ID_FIELD=votingapp_id
```

Comment:
Nope, just wanted to leave it as a "generic" value in the example file, since this isn't only for AGG. The `agg_id` and `cgg_id` fields are set as fillable in Northstar.